src/libs/actions/Report.js

```diff
@@ -140,8 +140,8 @@ function getChatReportName(fullReport, chatType) {
     if (isDefaultRoom({chatType})) {
         return `#${fullReport.reportName}${(isArchivedRoom({
             chatType,
-            stateNum: fullReport.stateNum,
```

Comment:
Out of curiosity, are we still passing `stateNum` from the back end, and if so is it being used anywhere in the front end?

Reply:
We're not. It comes from the back-end as `state`/`status`, but elsewhere in E.chat it's `stateNum`/`statusNum`